### DISCUSSION TITLE
Possibility of Compatibility with x86 Platform

### DISCUSSION THREAD

**Discussion by fangkaix** [bao-project/bao-hypervisor#88] [General]
Is there a possibility for ‘Bao’ to be compatible with the x86 platform in the future? If so, do you have any plans for such compatibility in the near future?

**Comment by josecm** [comment]
Although I'm not familiar with x86 in-depth, I would say that it's definitely a possibility! But while the idea of exploring x86 support has been discussed within our bao-project team, it is not currently on our near-term roadmap. Our current focus is directed towards the other end of the spectrum - more MCU-like architectures, such as Arm Cortex-R/M or Infineon's Tricore. Nevertheless, if an external party is interested in contributing x86 support, we would enthusiastically welcome their initiative and offer our full support.
